benchmark/vgg16_benchmark.cpp

```diff
@@ -50,14 +51,18 @@ int main(int argc, char** argv) {
       static_cast<float*>(model.get_buffer_handle(softmax_out_name));
     */
 
-    auto start = clock::now();
+    std::vector<std::chrono::microseconds> usecs;
 
-    // Run inference
-    model.run();
+    for (int i = 0; i < a.get<int>("iteration"); ++i) {
+        auto start = clock::now();
 
-    auto end = clock::now();
-    std::cout << std::chrono::duration_cast<std::chrono::milliseconds>(end -
-                                                                       start)
-                   .count()
-              << std::endl;
+        // Run inference
+        model.run();
+
+        auto end = clock::now();
+        usecs.push_back(std::chrono::duration_cast<std::chrono::microseconds>(end - start));
+    }
+
+    auto total_usec = std::accumulate(usecs.begin(), usecs.end(), std::chrono::microseconds());
```

Comment:
I wonder that `std::chrono::microseconds()` is specified always to be zero?

Reply:
I don't know, but

* https://en.cppreference.com/w/cpp/chrono/duration/duration says the default constructor is the default one and
* https://en.cppreference.com/w/cpp/chrono/duration says `std::chrono::microseconds` is `duration</*signed integer type of at least 55 bits*/, std::micro>`

so I think it's safe to assume the default value is zero. That said, I think it's good to use `zero` to make the intention clearer. I changed the default constructor to `zero`.

Reply:
It looks perfect :+1: